en/flight_controller/betafpv_beta75x.md

```diff
@@ -0,0 +1,33 @@
+# BetaFPV Beta75X 2S Brushless Whoop
+
+The [BetaFPV Beta75X](https://betafpv.com/products/beta75x-2s-whoop-quadcopter) is a very small quadrotor that can be flown indoors or outdoors, FPV or line-of-sight.
+
+![BetaFPV Beta75X](../../assets/hardware/betafpv_beta75x.jpg)
+
+## Where to Buy
+
+The *Beta75X* can be bought from a number of vendors, including:
+- [GetFPV](https://www.getfpv.com/beta75x-2s-brushless-whoop-micro-quadcopter-xt30-frsky.html)
+- [Amazon](https://www.amazon.com/BETAFPV-Beta75X-Brushless-Quadcopter-Smartaudio/dp/B07H86XSPW)
+
+In addition you will need an RC transmitter (make sure to select the matching receiver version), FPV goggles if you want to fly FPV, and a LiPo battery charger.
+
+## Flashing PX4
+
+The *Beta75X* comes preinstalled with Betaflight.
+
+To install the PX4 bootloader and firmware follow the [Omnibus F4 instructions](omnibus_f4_sd.md#betaflight_configurator) (this is the flight controller board on the *Beta75X*).
+
+> **Note** After following these instructions to install PX4, you can use *QGroundControl* for subsequent updates.
+
+## Connecting QGroundControl
+
+Once the firmware is flashed, you should be able to connect QGC via USB cable.
+
+Go through the initial setup, start with setting the [Airframe](../config/airframe.md) to *BetaFPV Beta75X 2S Brushless Whoop*, and continue with sensor calibrations and radio setup.
+
+## Video
```

Comment:
I added your video here. If you have a better one then great, but I don't think we need to wait on it.